ext/standard/file.c

```diff
@@ -1040,7 +1040,7 @@ PHPAPI PHP_FUNCTION(fgets)
 		efree(buf);
 	} else if (argc > 1) {
 		if (len <= 0) {
-			zend_value_error("Length parameter must be greater than 0");
+			zend_argument_value_error(2, "to be greater than or equal to 0");
```

Comment:
This is wrong, it should be strictly greater than 0 as the condition is triggered when ``len`` is equal to zero

Reply:
Yeah, I don't know why I couldn't pay attention to the operators... :)